### PR TITLE
Add tree creating

### PR DESCRIPTION
**Pull request by jakub-drak-sbahi**
image

### CODE REVIEW

File: web/src/app/tree/trees-list/trees-list.component.ts [hunk 1]
```diff
@@ -15,8 +15,12 @@ export class TreesListComponent implements OnInit {
   ) { }
 
   ngOnInit(): void {
+    this.onModelChange();
+  }
+
+  onModelChange() {
     this.service.getMyTrees().subscribe(result => {
```
Comment (BartoszChrostowski) [inline web/src/app/tree/trees-list/trees-list.component.ts:22]: I assume that after creating the tree, you are asking API for list?

**Review by BartoszChrostowski — CHANGES_REQUESTED** [review]
Make it the same height!!!!

File: web/src/app/tree/trees-list/trees-list.component.ts (hunk 1 above)
Comment (jakub-drak-sbahi) [reply web/src/app/tree/trees-list/trees-list.component.ts:22]: Exactly, parent component receives event from the child and calls this method

**Review by BartoszChrostowski — APPROVED** [review]

File: web/src/app/tree/trees-list/trees-list.component.ts (hunk 1 above)
Comment (BartoszChrostowski) [reply web/src/app/tree/trees-list/trees-list.component.ts:22]: U can just append the list with the newly created name. Whatever

**Comment by jakub-drak-sbahi** [comment]
> Make it the same height!!!!

Done

**Pull request merged**